tests/vmi_lifecycle_test.go

```diff
@@ -573,6 +573,52 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 				tests.WaitForSuccessfulVMIStart(newVMI)
 			})
+
+			It("[test_id:5275]VMIs with Bridge Networking shouldn't fail after the kubelet restarts", func() {
+				bridgeVMI := vmi
+				// Remove the masquerade interface to use the default bridge one
+				bridgeVMI.Spec.Domain.Devices.Interfaces = nil
+				bridgeVMI.Spec.Networks = nil
+				v1.SetDefaults_NetworkInterface(bridgeVMI)
+				Expect(bridgeVMI.Spec.Domain.Devices.Interfaces).NotTo(BeEmpty())
+				Expect(err).To(BeNil(), "Should submit VMI successfully")
+
+				By("starting a VMI with bridged network on a node")
+				bridgeVMI, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(bridgeVMI)
+
+				// Start a VirtualMachineInstance with bridged networking
+				nodeName := tests.WaitForSuccessfulVMIStart(bridgeVMI)
+
+				tests.VerifyDummyNicForBridgeNetwork(bridgeVMI)
+
+				By("restarting kubelet")
+				pod := renderPkillAllPod("kubelet")
+				pod.Spec.NodeName = nodeName
+				_, err = virtClient.CoreV1().Pods(tests.NamespaceTestDefault).Create(pod)
+				Expect(err).ToNot(HaveOccurred())
+
+				Eventually(func() error {
+					podList, err := virtClient.CoreV1().Pods(tests.NamespaceTestDefault).List(metav1.ListOptions{})
+					Expect(err).ToNot(HaveOccurred(), "Should list the pods")
+					for _, item := range podList.Items {
+						if strings.HasPrefix(item.Name, vmi.ObjectMeta.GenerateName) {
+							return nil
+						}
+					}
+					return fmt.Errorf("Associated pod for VirtualMachineInstance '%s' not found", bridgeVMI.Name)
+				}, 75, 0.5).Should(Succeed(), "Should find the bridgeVMI pod")
```

Comment:
Why do you need this verification? In this stage we don't even know kubelet was already re-started.
I think it is enough to verify the vm is running after you make sure kubelet was restarted.

Reply:
I added this to verify that kubelet is back up.  Will only creating a new VM verify that?

Reply:
I tested it locally and even when kubelet is down on one of the nodes I can see those node pods. So I don't thing querying the pod verifies kubelet is up.